internal/driver/commands.go

```diff
@@ -336,27 +336,28 @@ func listHelp(c string, redirect bool) string {
 }
 
 // browsers returns a list of commands to attempt for web visualization.
-// Commands which definitely will open a browser are prioritized over other
-// commands like xdg-open,  which may not open the javascript embedded SVG
-// files produced by the -web command in a browser.
 func browsers() []string {
 	var cmds []string
 	if userBrowser := os.Getenv("BROWSER"); userBrowser != "" {
 		cmds = append(cmds, userBrowser)
 	}
-	cmds = append(cmds, []string{"chrome", "google-chrome", "chromium", "firefox"}...)
 	switch runtime.GOOS {
 	case "darwin":
 		cmds = append(cmds, "/usr/bin/open")
 	case "windows":
 		cmds = append(cmds, "cmd /c start")
 	default:
+		// sensible-browser prioritized over xdg-open, so browser() command can
+		// be used on linux to open the .svg file generated by the -web command
+		// (the .svg file includes embedded javascript so is best viewed in a
+		// browser).
+		cmds = append(cmds, "sensible-browser")
 		if os.Getenv("DISPLAY") != "" {
+			// xdg-open is only for use in a desktop environment.
 			cmds = append(cmds, "xdg-open")
 		}
-		cmds = append(cmds, "sensible-browser")
 	}
-	return cmds
+	return append(cmds, []string{"chrome", "google-chrome", "chromium", "firefox"}...)
```

Comment:
I feel it's a bit weird that "sensible-browser" priority is different from these explicit browser commands. I think the point behind "sensible-browsers" is that sometimes this command may be missing (I think it's Debian / Ubuntu only?) and so we fall back to "chrome" etc. explicitly. Having anything in between "sensible-browser" and "chrome" and friends doesn't look right.

Reply:
Modified to only use chrome (and friends) on linux.